### ISSUE TITLE
Counter64 Support

### ISSUE THREAD

**Issue by LemonIceScream** [cdevr/WapSNMP#17]
Hello,

The library doesn't seem to support 64-bit counters such as those in the ifXTable: http://tools.cisco.com/Support/SNMP/do/BrowseOID.do?local=en&translate=Translate&objectInput=1.3.6.1.2.1.31.1.1.1

I have taken a look at DecodeSequence in ber.go and have added the following to the BERType table to make it work:

```go
        case AsnCounter64:
            val, err := DecodeUInt(berValue)
            if err != nil {
                return nil, fmt.Errorf("error decoding integer %v: %v", berValue, err)
            }
            result = append(result, Counter64(val))
```

**Comment by cdevr** [comment]
Thanks. I will include something similar in the library soon.

**Comment by cdevr** [comment]
This was actually included in 53d49f1d and is solved. Forgot to close the bug.
